ttfinfo is required

## Description
```

  Error: Cannot find module 'ttfinfo'
      at Module._resolveFilename (internal/modules/cjs/loader.js:602:15)
      at Function.Module._resolveFilename (/Users/akira/.akira/src/github.com/h3poteto/roma-desktop/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
      at Function.Module._load (internal/modules/cjs/loader.js:528:25)
      at Module.require (internal/modules/cjs/loader.js:658:17)
      at require (internal/modules/cjs/helpers.js:20:18)
      at Object.<anonymous> (/Users/akira/.akira/src/github.com/h3poteto/roma-desktop/node_modules/system-font-families/dist/main.js:17:16)
      at Object.<anonymous> (/Users/akira/.akira/src/github.com/h3poteto/roma-desktop/node_modules/system-font-families/dist/main.js:261:3)
      at Module._compile (internal/modules/cjs/loader.js:711:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:722:10)
      at Module.load (internal/modules/cjs/loader.js:620:32)
```

## How To Reproduce
1. Clone repository and npm install in clean machine
2. `npm run dev`

## Your Environment
 - OS: MacOS
 - Whalebird Version: 2.6.3
 - Instance: all

--- a/package.json
+++ b/package.json
@@ -209,6 +209,7 @@
     "style-loader": "^0.23.1",
     "stylelint": "^9.9.0",
     "stylelint-config-standard": "^18.2.0",
+    "ttfinfo": "^0.2.0",
     "url-loader": "^1.1.2",
     "vue-html-loader": "^1.2.4",
     "vue-loader": "^15.2.4",

